packages/suite/src/views/dashboard/AssetsView/AssetCard/AssetCard.tsx

```diff
@@ -186,17 +190,25 @@ export const AssetCard = ({
                 />
             )}
             {!isTestnet(symbol) && (
-                <Card>
+                <Card data-testid="@dashboard/asset/bottom-info">
                     <Row justifyContent="space-between" gap={spacings.md}>
-                        <InfoItem label={<Translation id="TR_EXCHANGE_RATE" />} flex="0">
+                        <InfoItem
+                            data-testid="@dashboard/asset/exchange-rate"
+                            label={<Translation id="TR_EXCHANGE_RATE" />}
+                            flex="0"
+                        >
                             <PriceTicker symbol={symbol} />
                         </InfoItem>
-                        <InfoItem label={<Translation id="TR_7D_CHANGE" />} flex="0">
+                        <InfoItem
+                            data-testid="@dashboard/asset/week-change"
+                            label={<Translation id="TR_7D_CHANGE" />}
+                            flex="0"
+                        >
                             <TrendTicker symbol={symbol} />
                         </InfoItem>
                         <CoinmarketBuyButton
                             symbol={symbol}
-                            data-testid={`@dashboard/assets/grid/${symbol}/buy-button`}
```

Comment:
Seems like no other tests use this one. So I updated it to be consistent with the new one,